PerfNext/.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules
 /bin/
 /.apt_generated/
-perfnext_passwords
+credentials.json
+.README.md.html
+.DS_Store
```

Comment:
Please add a new line at the end of this file

Reply:
Done!